src/op_analytics/datapipeline/models/templates/erc20_contracts/erc20_contract_creation.sql.j2

```diff
@@ -0,0 +1,14 @@
+SELECT
+  ct.*
+  , 'erc20' AS token_standard
+  , sc.token_type
+FROM {{ contract_creation_traces }} AS ct
+LEFT JOIN {{ superchainerc20 }} AS sc
```

Comment:
I wonder if it makes sense to join to token standards here, or as a later view?

Since we can add more of these over time.

Reply:
yea indeed, currently keeping it flexible as a separate view, but i wonder how many that'd be. Or maybe in the future we just keep creating different token types in template and then left join all in dbt instead of the current pipeline. Also cc: @lithium323 if you have any thought here